js/simulation.coffee

```diff
@@ -8,7 +8,7 @@
 
 
 if SIM_NAME of ALL_DATA
-  build(ALL_DATA[SIM_NAME].meta, SIM_NAME, CODES_DATA, CHART_DATA, AXES_NAMES, REPO)
+  build(ALL_DATA[SIM_NAME].meta, SIM_NAME, CODES_DATA, CHART_DATA, AXES_NAMES, REPO, BENCHMARK_DATA)
```

Comment:
`_includes/coffee/simulation.coffee:704` breaks elements of the list onto separate lines. That's easier to read than this grouping of all on one line.

Reply:
Done